open_spiel/games/dou_dizhu/dou_dizhu_utils_test.cc

```diff
@@ -0,0 +1,313 @@
+// Copyright 2022 DeepMind Technologies Limited
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <iostream>
+#include <cstring>
+#include "open_spiel/abseil-cpp/absl/types/optional.h"
+#include "open_spiel/abseil-cpp/absl/strings/str_format.h"
+#include "open_spiel/abseil-cpp/absl/strings/string_view.h"
+#include "open_spiel/spiel.h"
+
+#include "open_spiel/games/dou_dizhu/dou_dizhu_utils.h"
+
+
+namespace open_spiel{
+namespace dou_dizhu{
+
+
+
+void SingleRankHandTest(){
+  std::array<int, kNumRanks> hand1{};
+  // 999
+  hand1[6] = 3;
+  int action_id1 = SingleRankHandToActionId(hand1);
+  // std::cout << "Begin test for transforming hands to Ids" << std::endl;
+  // std::cout << absl::StrFormat("Hands: %s", FormatSingleHand(hand1)) << std::endl;
+  // std::cout << action_id << std::endl;
+  // std::cout << FormatSingleHand(SingleRankHand(action_id)) << std::endl;
+  SPIEL_CHECK_EQ(FormatSingleHand(SingleRankHand(action_id1)), "999");
+
+  std::array<int, kNumRanks> hand2{};
```

Comment:
I don't think you need the {}?

Reply:
The same above. It will be more convenient if {} is used and then the values are initialized to 0